### DISCUSSION TITLE
sqlite3.OperationalError: locking protocol

### DISCUSSION THREAD

**Discussion by keshavadk** [cvat-ai/cvat#4276] [Q&A] [answered]
When i run the CVAT command for running the backend service i.e  python manage.py runserver 7000 , i get the error, 
**sqlite3.OperationalError: locking protocol**

which is attached in the image below.

Any hint ,please?

![sql_lite_error](https://user-images.githubusercontent.com/16021947/152136921-4d85fec8-3960-48aa-aa1a-d6973d3ef677.JPG)

**Comment by azhavoro** [comment, marked as answer]
Hi, try to clear `<cvat_root>/data/cache` folder

**Reply by keshva23** [reply]
Ok i will do that, Thank you @azhavoro .

**Reply by keshva23** [reply]
@azhavoro Do we need to clear cache from <cvat_root>/data/cache each time during development . ?

Because it is generating 'folders' inside 'cache' again after sometime.

**Reply by azhavoro** [reply]
> @azhavoro Do we need to clear cache from <cvat_root>/data/cache each time during development . ?
> 
> Because it is generating 'folders' inside 'cache' again after sometime.

Usually that is unnecessary

**Reply by keshva23** [reply]
ok Thank you @azhavoro
